### ISSUE TITLE
Wrong pb file after convertion

### ISSUE THREAD

**Issue by MoonBunnyZZZ** [amir-abdi/keras_to_tensorflow#60]
My model contain custom optimizer and loss function.When loading model in the keras_to_tensorflow.py,I use custom_objects={'xxx':xxx} in the load_model function.

After convertion,I find ,at least,the input and output does not match my algorithm.While the input shape should be [x,x,x,3] and the output shape [x,x,x,19]+[x,x,x,38](because my net has two branch),the input  become two [?, ?, ?, 3],[?, ?, ?, 38] and the output shape is [?, ?, ?, 38] in the pb file. @amir-abdi
